kafka-rest/src/test/java/io/confluent/kafkarest/integration/SchemaRegistrySaslInheritTest.java

```diff
@@ -30,28 +30,30 @@
 import javax.ws.rs.core.Response;
 import org.apache.kafka.clients.consumer.ConsumerRecord;
 import org.apache.kafka.common.security.auth.SecurityProtocol;
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.RuleChain;
-import org.junit.rules.TestRule;
-import org.junit.runner.RunWith;
-import org.junit.runners.JUnit4;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
 
-@RunWith(JUnit4.class)
```

Comment:
Should you tag this as integration test, since it doesn't inherit from ClusterTestHarness?

Reply:
Most integration tests inherit from ClusterTestHarness, so now they get the annotation from inheritance. The fact this one missed the annotation is a bug. I never realized we needed it. Now we do. I think we should have fixed it. :(

Reply:
We can do this and other ones missing the annotation in their own PR.